osbs/api.py

```diff
@@ -653,6 +653,17 @@ def _check_labels(self, repo_info):
         except KeyError:
             pass
 
+        try:
+            _, version_value = labels.get_name_and_value(Labels.LABEL_TYPE_VERSION)
+            if (version_value and
+                    any(denied in version_value for denied in VERSION_LABEL_FORBIDDEN_CHARS)):
+
+                logger.error("version label '%s' contains not allowed chars : %s", version_value,
+                             VERSION_LABEL_FORBIDDEN_CHARS)
+                raise OsbsValidationException("version label contains not allowed chars")
```

Comment:
Can we specify which one?

Reply:
+1 for `denied` instead of `VERSION_LABEL_FORBIDDEN_CHARS`

Reply:
sure

Reply:
I meant in that exception, so it's shown to a user directly

Reply:
added there as well :)